collect/collect.go

```diff
@@ -371,30 +355,7 @@ func (i *InMemCollector) sendAfterRootDelay(trace *types.Trace) {
 		sendDelay = 2
 	}
 	dur := time.Duration(sendDelay) * time.Second
-	go i.pauseAndSend(dur, trace)
-}
-
-// pauseAndSend will block for pause time and then send the trace.
-func (i *InMemCollector) pauseAndSend(pause time.Duration, trace *types.Trace) {
-	select {
-	case <-time.After(pause):
-		trace.SendOnce.Do(func() {
-			// TODO fix FinishTime to be the time of the last span + its duration rather
-			// than whenever the timer goes off.
-			trace.FinishTime = time.Now()
```

Comment:
`pauseAndSend` added a `FinishTime` to the trace. Should this be happening in the new code as well?

Reply:
ooh good catch, I'll look into where we can do that now. It's only used for stats AFAICT